Sources/Controllers/Others/OAWebViewController.h

```diff
@@ -11,6 +11,8 @@
 @interface OAWebViewController : OABaseWebViewController
 
 @property NSString *urlString;
+@property BOOL *isCssOverriding;
+@property BOOL *isDarkModeSupported;
```

Comment:
omg) BOOL cannot be a reference type. @property (nonatomic, assign) BOOL myBool;